### PR TITLE
QA-1426: Updated README with resiliency tests section.

### PR DESCRIPTION
**Pull request by ichengchang**
The updated README provides background information on resiliency tests. The requirements for setting up and running resiliency tests using Test Runner Framework are documented as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,11 +86,31 @@ The application specification includes (this section applies to resiliency test
     * componentLabel
     * apiComponentLabel
 
-### Running Resiliency Tests Through Test Runner Library
-Test Runner Library provides the underlying framework for running resiliency tests within namespaces. The whole process involves only a few simple setup and configuration steps described below.
+#### Running Resiliency Tests
```
[the remark this answers is not in the document]
Comment (ichengchang) [reply README.md:226]: Moved resiliency test section under `Execute a test run` section.

File: README.md [hunk 2]
```diff
@@ -85,11 +86,31 @@ The application specification includes (this section applies to resiliency test
...
+Test Runner Framework supports resiliency tests in addition to Integration, Performance, and Connected tests.
+As a premier genomic platform for biomedical research, the [*Terra.Bio*](https://terra.bio) migration to *PaaS* or Cloud infrastructures (GCP, AWS, Azure) to create the new *MCTerra* platform, is a key step to advance the field of genomic science.
+Without the migration and data crunching capability that comes with the migration, it would be difficult for researchers to develop, experiment with and evaluate next generation *SOTA* algorithms for genomic computing at scale.
+Our goal is to ensure that the *MCTerra* platform continues to function and scale properly as demand continues to increase. 
+
+Given the above context, *Containerization* has become the *defacto* go to paradigm for managing shared cloud resources the *MCTerra* platform requires during its lifecycles.
```
[the remark this answers is not in the document]
Comment (ichengchang) [reply README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -85,11 +86,31 @@ The application specification includes (this section applies to resiliency test
...
+Resiliency tests can trigger additional logging in targeted *MCTerra* service components for further analysis.
+
+***What is a resiliency test?***
+
+Test Runner provides the framework to target containerized applications with specific loads while at the same time control the cloud resources given to the containerized applications.
+During a resiliency test flight, the framework spawns concurrent threads to scale cloud resources up and down while delivering load to the target *MCTerra* service components at scale according to user specifications.
+Resiliency tests can be integrated with a CI/CD pipeline such as GitHub Action Workflows, or they can be run locally for debugging purpose.
+
+Test Runner Framework supports resiliency tests on a ***namespaced*** test environment. The following discussion assumes a valid namespace already exists in a Kubernetes cluster.
+Please refer to [Creating the namespace](https://github.com/DataBiosphere/terra/blob/iv-1331/docs/dev-guides/personal-environments.md#step-3-creating-the-namespace) for more details about namespace creation in *MCTerra*.
```
[the remark this answers is not in the document]
Comment (ichengchang) [reply README.md]: Updated link to master branch.

File: README.md [hunk 4]
```diff
@@ -98,27 +119,115 @@ The `setup-k8s-testrunner.sh` script also carries out the following functions:
...
-The second argument is the Terra namespace (without the `terra-` prefix).
+The second argument is the Terra namespace (without the `terra-` prefix).Without 
 
 The third argument is just some text to describe the application itself.
 ```shell script
 $ ./setup-k8s-testrunner.sh gke_terra-kernel-k8s_us-central1-a_terra-integration zloery workspacemanager
 ```
 
-Once the script above ran successfully, the namespace is ready for resiliency testing through the Test Runner Framework.
+In summary, the script automatically templates in the variables `__KUBECONTEXT__, __NAMESPACE__, __APP__` based on the 3 arguments presented above and create the necessary namespace objects in Kubernetes that enables Test Runner to control the namespace.
```
[the remark this answers is not in the document]
Comment (ichengchang) [reply README.md]: Fixed.

**Review by davidangb — APPROVED** [review]
looks fine to me, I did not try following any of the instructions here, I only read through it

**Pull request merged**